app/routes/telemetry/index.js

```diff
@@ -47,9 +49,19 @@ const CORE_METRICS = {
     "hydrocatFluorescence",
     "hydrocatTurbidity",
     "hydrocatPH",
-  ],
-  Hydrocycle: ["CAPO4"],
-  SUNA: ["sunaNitrateMicroMol"],
+    "hydrocatStartTime"],
+  Hydrocycle: [
+  "CAPO4",
+  "QCflag",
+  "Bubbleflag",
+  "COVflag",
+  "Lowsigflag",
+  "OoRflag",
+  "Mixingflag",
+  "Calflag",
+  "Hydrocyclestart"]
+  SUNA: ["sunaNitrateMicroMol",
+  "sunaStart"],
   MetData: [
```

Comment:
why no start time for metdata and par? Ideally, the field would be consistent across tables like the `timeStamp` and then we could programmatically insert it into the queries on the service side instead of here on the API definition.

Reply:
Initially for our purposes I did not include the new start times for these measurements as they should be instantaneous, with the existing timestamp matching the start time but I agree that the new column should be added for these tables for ERDDAP. I am working on adding that now, and have also removed the "Time" portion of the Hydrocat column name.